### ISSUE TITLE
Nothing shows

### ISSUE THREAD

**Issue by Niruzoid** [pranavgade20/FabricHax#14]
Nothing loads in other than chests and animals, all other blocks and everything doesn't work, screenshot added
1. Your computer's Operating System, and version
    Windows 10 Home, 21H1
2. Java version (open a terminal and type java -v)
   Version 8, Update 291
3. What version of MineCraft you are running
   1.17
4. What you tried that caused an error
   Joining a world or server.
5. Any other info that can up pinpoint the error
![image](https://user-images.githubusercontent.com/67180137/124557090-724ea600-de7c-11eb-8956-7a9dfa9b2acc.png)
![image](https://user-images.githubusercontent.com/67180137/124557256-a32edb00-de7c-11eb-921e-a88c471699fc.png)

**Comment by pranavgade20** [comment]
Are you sure that this mod is what is causing rendering issues? Could you try to delete this mod and see if it works?

**Comment by Niruzoid** [comment]
> Are you sure that this mod is what is causing rendering issues? Could you try to delete this mod and see if it works?

I already tried this and it was the problem, if i dont have the mod installed, it works perfectly fine.

**Comment by pranavgade20** [comment]
Okay, thank you for the report! I'll try to fix this as soon as I can :)

**Comment by Niruzoid** [comment]
Cool!

**Comment by pranavgade20** [comment]
It appears to be a problem with the bundled jvm. Until I find a fix, you can try installing this https://jdk.java.net/16/

**Comment by pranavgade20** [comment]
Could you try using the latest release? https://github.com/pranavgade20/FabricHax/releases/tag/v1.3.3

Also delete the fabrichax.config file from your mods folder

**Comment by Niruzoid** [comment]
yo, getting a crash error
![image](https://user-images.githubusercontent.com/67180137/125180097-6d517400-e239-11eb-8c5b-60f014764906.png)
https://pastebin.com/Gi5j0YYM (crash log)

**Comment by blueForestIcarus** [comment]
I get the same thing

**Comment by pranavgade20** [comment]
Yup it's the same go for me as well, but I think this is an issue with fabric API itself, so I'm waiting for an update  which will hopefully fix this. Otherwise, I'll have to work on removing fabric API as a dependency

**Comment by niruvibes** [comment]
Did this ever get fixed?

**Comment by pranavgade20** [comment]
Fixed it :sweat_smile: https://github.com/pranavgade20/FabricHax/releases/tag/v1.3.5
